packages/react/src/Section/Section.tsx

```diff
@@ -0,0 +1,175 @@
+import React, {forwardRef, PropsWithChildren, useCallback, useMemo} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/section/section.css'
+import styles from './Section.module.css'
+
+export const SectionPaddingVariants = ['none', 'condensed', 'normal', 'spacious'] as const
+export const SectionBackgroundColors = ['default', 'inset', 'subtle', 'overlay'] as const
```

Comment:
WDYT about dropping the inset and overlay colors? I think we might drop those with v8, and we only ever use default and subtle. I've also excluded those in my BreakoutBanner component for same reasons.

Reply:
Let's go for it. 👍